### DISCUSSION TITLE
My request was rejected but I have no idea about that...

### DISCUSSION THREAD

**Discussion by pai233** [community/community#31014] [GitHub Education]
Hello, I'm a middle school student from mainland China. I made a educational discount request this morning and it was rejected. I uploaded a certificate from my school because the most of Chinese middle school don't give the ID cards to the students. I don't have anything like ID card. I have no idea about that. I hope someone can give me some ideas.

There is my Chinese name on the certificate from my school, but I use my English name on Github.

**Comment by supengepeng** [comment]
我也是中国大陆学生，可以用学信网的学籍证明，用Google浏览器翻译成英文。

**Comment by pai233** [comment]
因为我是初中生，用不了学信网，我后面实拍了那张奖状并修改了我的个人资料，现在已经通过了

supengpeng ***@***.***> 于 2022年8月29日周一 17:07写道：
